### PR TITLE
Updated guidelines

### LINKED ISSUE #334: Update landing page guidelines with actual text
I don't want to send the website to people and they get confused by our All Star by Smashmouth lorem-ipsum guidelines lol

Could probably use basically the same text we use for the README update (whenever that gets made)

### PR DESCRIPTION
**Pull request by gannonprudhomme**
## Description

Updates the guidelines!

Note that the css I have here is temporary, but if we commit to using it I'll probs change it to be it's own file

## Screenshots

|With `<br />'s`|With `<p>`'s (current)|
|--|--|
|chrome_9wf9tKAfuo|chrome_PBE9wTd1ip|

chrome_1Oa8r5GDsP

## Related tasks

Closes #334

### CODE REVIEW

File: autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+    <div style={{
+      gap: 50,
+    }}
+    >
+      <p>
+        Simply select a term, pick your courses, mark off when you&apos;re not available, we&apos;ll
+        generate schedules for you!
+      </p>
+      <p>
+        Rev Registration is currently a work-in-progress, so please mind any bugs or mishaps that
```
Comment (firejake308) [inline autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx:17]:
```suggestion
        This site is currently a work-in-progress, so if you run into any bugs or have any suggestions, feel free to file an issue in <a>our GitHub</a> or send us <a>an email</a>.
```

File: autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
-    title="Guidelines"
+    title="Welcome to Rev Registration!"
   >
-    {bodyText}
+    <div style={{
+      gap: 50,
+    }}
+    >
+      <p>
+        Simply select a term, pick your courses, mark off when you&apos;re not available, we&apos;ll
```
Comment (firejake308) [inline autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx:13]:
```suggestion
        Rev Registration is an automatic schedule generator for Texas A&M. Simply select a term, pick your courses, mark off when you&apos;re not available, we&apos;ll
```

**Review by firejake308 — CHANGES_REQUESTED** [review]

File: autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx (hunk 2 above)
Comment (gannonprudhomme) [reply autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx:13]: Didn't we agree on `automatic class scheduler`?

**Comment by gannonprudhomme** [comment]
Here's how it looks now: 
chrome_p4XxkSUKF8

File: autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx (hunk 2 above)
Comment (ryan-conn) [reply autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx]: Yeah I'm pretty confident we did

Comment (firejake308) [reply autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx]: oops, my bad, I wrote down the wrong version. `automatic class scheduler` is perfect, no changes needed

File: autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
+      <p>
+        Rev Registration is an automatic class scheduler for Texas A&amp;M. Simply select a term,
+        pick your courses, mark off when you&apos;re not available, we&apos;ll generate schedules
+        for you!
+      </p>
+      <p>
+        This site is currently a work-in-progress, so if you run into any bugs or have any
+        suggestions, feel free to file an issue on&nbsp;
+        <a href="https://github.com/aggie-coding-club/Rev-Registration/issues/new">our GitHub</a>
+        , or by sending us an email at:&nbsp;
```
Comment (firejake308) [inline autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx:21]:
```suggestion
        , or send us an email at:&nbsp;
```

**Review by firejake308 — CHANGES_REQUESTED** [review]
Grammar

**Review by ryan-conn — APPROVED** [review]

**Pull request merged**